share/spice/sports/mlb/games/sports_mlb_games.js

```diff
@@ -0,0 +1,175 @@
+            // in a desktop environment, or perhaps in a multiple-call
+            // scenario where there isn't a 'more relevant' game.
+            if (!DDG.device.isMobile || !data.most_relevant_game_id) {
+                return data.games; 
+            } else if (DDG.device.isMobile && data.most_relevant_game_id) {
+                var foundRelevant = false,
+                    games = [],
+                    i = 0;
+                // for historical games on the first call, 
+                // only add the most relevant game and one previous
+                // (if available)
+                for (; i < data.games.length; i++) {
+                    if (data.games[i].id === data.most_relevant_game_id) {
+                        // most relevant first
+                        games.push(data.games[i]);
+                        // one historical game if the layout allows for it
+                        if (!DDG.device.isMobileLandscape() && data.games[i-1]) { games.push(data.games[i-1]); }
+                        foundRelevant = true;
+                    } else {
+                        if (foundRelevant) {
+                            games.push(data.games[i]);
+                        } else {
+                            continue;
+                        }
+                    }
+                }
+                return games;
+            }
+        },
+        
+        getRelativeDay = function(dateStr) {
+            // moment.js returns 'rounded-down' date diffs
+            // based on hours, making it useless for 
+            // directly determining relative calendar days.  
+            //
+            // Thus, this function just uses the diff to determine
+            // if the game is within the same week, and determines
+            // today/tomorrow/yesterday based on the day of the week.
+            //
+            // This gets around the sticky end-of-month issues that
+            // so many relative date functions encounter. :)
+            
+            // Weekdays are integers representing Sunday (0) 
+            // through Saturday (6).
+            var today = moment(),
+                date = moment(DDG.getDateFromString(dateStr)),
+                diff = today.diff(date, 'days'),
+                dayDiff = today.weekday() - date.weekday();
+            
+            if (diff > -2 && diff < 7) {
+                switch(dayDiff) {
+                    case 0:
+                        return l("Today");
+                        break;
+                    case -1:
+                    // fall through
+                    case 6: // Saturday (6) - Sunday (0)
+                        return l("Tomorrow");
+                        break;
+                    case 1:
+                        return l("Yesterday");
+                        break;
+                    default:
+                        return false;
+                        break;
+                }
+            }
```

Comment:
yea, my bad, should be `isSame`, was typing from memory. Did it work though? Can we make the change?